packages/flame_network_assets/README.md

```diff
@@ -0,0 +1,25 @@
+# flame_network_assets
+
+<!-- markdownlint-disable MD013 -->
+<p align="center">
+  <a href="https://flame-engine.org">
+    <img alt="flame" width="200px" src="https://user-images.githubusercontent.com/6718144/101553774-3bc7b000-39ad-11eb-8a6a-de2daa31bd64.png">
+  </a>
+</p>
+
+<p align="center">
+Adds network images support to <a href="https://github.com/flame-engine/flame">Flame</a>.
+</p>
+
+<p align="center">
+  <img src="https://github.com/flame-engine/flame_network_image/workflows/Lint/badge.svg?branch=master&event=push" alt="Test" />
+  <a title="Discord" href="https://discord.gg/pxrBmy4" ><img src="https://img.shields.io/discord/509714518008528896.svg" /></a>
```

Comment:
why not the 4 standard badges? (I know the package hasnt been released yet if you add as soon as it is it will work)